types/empty-dir/empty-dir-tests.ts

```diff
@@ -7,6 +7,9 @@ emptyDir('./', (err, isEmpty) => {
     isEmpty;
 });
 
+// $ExpectType boolean
+emptyDir.sync(["./test", "./array"]);
```

Comment:
Please also add an analogous test for the non-sync version.

Reply:
Added on line 13